### ISSUE TITLE
neurons in the 3D not constrained to grey matter (in 2D they are)

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#185]
From the client: The neuronal locations in the 3D viewer were not constrained to the Grey matter, while they were properly constrained in the 2D viewer

awaiting data and snaphots

**Comment by neuroBazzi** [comment]
@afonsobspinto they shared the data with us: it's  labeled V2b_PKD2L1 in this folder: 
https://drive.google.com/drive/folders/1-5dhTs-IsejBu_mLGSgDwAyEHc-VoTaa

**Comment by zsinnema** [comment]
I've uplaoded the dataset to the dev server and I notived this:
![image](https://user-images.githubusercontent.com/124044/183028459-bf119a66-7a9a-418f-9e97-7d2e9ffc9942.png)

couple of things here:
* the "scaling" of the neurons in the 3D viewer is not the same as in the 2D viewer
* the position of the "ring" of neurons in the 3D viewer is out of the center, in the 2D viewer it's in the center

**Comment by neuroBazzi** [comment]
@zsinnema agreed, it seems another manifestation of the same issue. also the three neurons that are at the edge of the grey matter in 2D are outside of it in the 3D. 
another question for @afonsobspinto, the ring of neurons seems a pretty unusual setting. Could it be that the coordinates of the edges of the Central Canal became neurons? the central canal should be more or less these and in that position. This would be a different issue from the fact that there is mismatch between the 2D/3D

**Comment by zsinnema** [comment]
@neuroBazzi cc: @afonsobspinto imo the issue here is the scaling and centering....

would be nice if we have a picture of the correct positioned neurons or a description how we can see them in e.g. Napari

**Comment by neuroBazzi** [comment]
@zsinnema ok I will ask Stephen Lenzi (I will also ask Matteo to add him to this repo so we can tag him in the future)

**Comment by zsinnema** [comment]
Hi @neuroBazzi any news on this?

**Comment by neuroBazzi** [comment]
@zsinnema not yet, Stephen is off. Anything we can do without his help?

**Comment by afonsobspinto** [comment]
I'm thinking that the issue in the 3D viewer is somehow related with the meshes. That's my take after rolling out problems in the InstancedMesh logic by doing the cells positioning in a visualizer independent of the application. Here we just simply draw spheres in the coordinates we read from the transformed_cells and they are still displaced:
![image](https://user-images.githubusercontent.com/19196034/186188608-17745862-aac8-409d-8766-84db7b751442.png)
In the process of testing I noticed that the latest meshes I have are actually out of scale with the coordinates in the transformed_cells file:
![image](https://user-images.githubusercontent.com/19196034/186189120-3c92cabf-6375-4415-9e7d-5ea32e4c238e.png)
I shared this with @stephenlenzi, plus I also mentioned to him the sort of 'ring' structure we get from 'V2b_PKD2L1' but that's for sure a different issue (probably to keep track on cordmap)
